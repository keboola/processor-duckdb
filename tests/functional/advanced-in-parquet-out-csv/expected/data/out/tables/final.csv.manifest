{"incremental": false, "write_always": false, "delimiter": ",", "enclosure": "\"", "column_metadata": {"id": [{"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}], "name": [{"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": false}]}, "columns": ["id", "name"]}