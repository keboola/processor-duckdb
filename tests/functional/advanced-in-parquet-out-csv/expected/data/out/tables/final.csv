id,name
"1","fruit"
"2","vegetable"
"3","beverages"
"4","chocolate"
"5","beer"
"6","wine"
